lua/nvim-treesitter/textobjects/select.lua

```diff
@@ -9,7 +9,7 @@ local ts_utils = require'nvim-treesitter.ts_utils'
 local M = {}
 
 function M.select_textobject(query_string)
-  local bufnr, textobject = shared.textobject_at_point(query_string)
+  local bufnr, textobject = shared.textobject_at_point(query_string, 'trynext')
```

Comment:
We though about adding this when this feature was introduced. Can you make it an optional?

I will also out-factor textobjects soon.

Reply:
> Can you make it an optional?

Do you mean an option in the config module?